batch/batch/server/server.py

```diff
@@ -81,15 +81,29 @@ def make_logger():
 
 class JobTask:  # pylint: disable=R0903
     @staticmethod
-    def copy_task(job_id, task_name, files):
-        if files:
-            container = kube.client.V1Container(image='alpine',
-                                                name=task_name,
-                                                command=['echo', 'hello'])
-
-            spec = kube.client.V1PodSpec(containers=[container],
-                                         restart_policy='Never')
-
+    def copy_task(job_id, task_name, files, copy_service_account_name):
+        if files is not None:
+            assert copy_service_account_name is not None
+            authenticate = 'gcloud -q auth activate-service-account --key-file=/gcp-sa-key/key.json'
+            copies = ' & '.join([f'gsutil cp {src} {dst}' for (src, dst) in files])
```

Comment:
The other option I looked into was using the `-m` option and giving `cp` a list of files to copy with `-I`. But that's not going to work bc it assumes the same destination directory.